tests/build_/test_build_request.py

```diff
@@ -2791,20 +2791,26 @@ def test_render_resolve_composes(self, odcs_insecure, additional_params, odcs_op
         plugin_args = plugin_value_get(plugins, plugin_type, plugin_name, 'args')
         assert plugin_args == expected_plugin_args
 
-
-    @pytest.mark.parametrize('enabled', (True, False))
-    def test_remove_resolve_composes(self, enabled):
+    @pytest.mark.parametrize(('odcs_url', 'yum_repo', 'raised'), [
```

Comment:
Nitpick - `yum_repos` would be a better name

Reply:
ok